### ISSUE TITLE
Reload destinations on HTTP tokens change

### ISSUE THREAD

**Issue by azhard4int** [jitsucom/jitsu#865]
## Description
A short description of the bug and its impact

### Problem

We are using Jitsu HTTP tokens endpoint to create the client id, secret, etc for each of our customers. Right now, Jitsu does not reload when the HTTP tokens are reloaded, leading to this error: No destination is configured for token 

```
2022-03-18 16:34:39 [INFO]: ✅ New resource [authorization] has been loaded
2022-03-18 16:34:57 [WARN]: No destination is configured for token ["UMTfG4TFwS"] (or only staged ones). Event: 
```

The destination mapping stays in the previous state.

## Steps to reproduce

1. Create an HTTP endpoint for authorization resources. 
2. Add a new token and send data through Jitsu Pixel.
3. It will raise an error as below

```
2022-03-18 16:34:39 [INFO]: ✅ New resource [authorization] has been loaded
2022-03-18 16:34:57 [WARN]: No destination is configured for token ["UMTfG4TFwS"] (or only staged ones). Event: 
```

# Expected behavior

The destinations should be reloaded so that there is no staged errors when the new tokens are added.

## Actual behavior

What's happening now and how it's different from actual behavior

The destinations are not reloading at the moment. We have to restart Jitsu to accept the events.

**Comment by viggin543** [comment]
you have this key in redis `configs#meta#last_updated`
![image](https://user-images.githubusercontent.com/6231756/159328250-9ceb7565-ed29-4a6f-bc99-d15eada57912.png)

update it with latest timestamp for the config you edited and it will cause a reload.

**Comment by vklimontovich** [comment]
We have API since `1.40`, so you don't really need to change Redis structures: https://jitsu.com/docs/other-features/jitsu-api

**Comment by absorbb** [comment]
Fixed in [1.41.1](https://github.com/jitsucom/jitsu/releases/tag/v1.41.1)
